src/hooks/prerun/install-prerun.js

```diff
@@ -1,10 +1,19 @@
 /* eslint-disable no-console */
 const chalk = require('chalk');
 
-const UPDATE_ALERT = `Warning: Use '${chalk.bold('npm update twilio-cli')}' for npm based installations. `;
+const urlUtil = require('../../services/hyperlink-utility');
+
+const TWILIO_CLI_DOCS_LINK = 'https://www.twilio.com/docs/twilio-cli/quickstart';
```

Comment:
Can use a short link(https://twil.io/cli) so it's neat and shows completely even on a small window?

Reply:
Great point! Changes made